express/experiments/ccx0027/blocks/quick-action1/quick-action.js

```diff
@@ -38,6 +41,17 @@ function createMockQuickAction() {
   + '<div class="quick-action-tag-container"><div class="quick-action-tag"><img class="icon icon-checkmark" src="/express/icons/checkmark.svg" alt="checkmark"></div>No credit card required</div>'
   + '</div> <input id="mock-file-input" type="file" accept="image/jpeg,image/png">'
   + '</div>';
+  container.append(ele);
+  container.append(createLegalCopy());
+  return container;
+}
+
+function createLegalCopy() {
+  const ele = document.createElement("div");
+  ele.className = "quick-action-legal-copy";
```

Comment:
*nitpicking: single quotes, as well as line above
```suggestion
  ele.className = 'quick-action-legal-copy';
```